emulator/clients/README.md

```diff
@@ -0,0 +1,23 @@
+# Emulator clients
+
+The emulator has the `--client` command line option, which will initialize a server on the client that listens for incoming connections. The clients transmit data to the server with information on how to set external pins. 
+
+## How it works
+
+The client transmits one or several `struct PinEvent` which contains information about when an event shall occur on a pin and whether it should be high or low. E.g., setting a pin high and immediately low again can be done by sending two `struct PinEvent` which sets that pin high and low again immediately after.
```

Comment:
The semantics of the interface could be explained a little better I think. You could show the struct here. Especially what does the `in_n_cycles` field mean? It seems to  me that it means one of two things:
A) Number of cycles until the event should occur relative the current clock cycle of the emulator
B) Number of cycles until the event should occur relative the time of the last event in the linked list of pending events

Reply:
Good suggestion, I've added B) now.